.travis.yml

```diff
@@ -1,18 +1,16 @@
 language: python
 sudo: required
 dist: trusty
-python: '3.4'
+python: '3.5'
```

Comment:
Why not 3.6?

Reply:
Probably should be going forward but kept it inline with the runtime.txt for development